components/Modal/ClaimModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect, MouseEvent } from 'react';
+import { useAuthContext, useModalContext } from '../Provider';
+import {
+  Background,
+  ModalBox,
+  Section,
+  CloseIconContainer,
+  Title,
+  Description,
+  Row,
+  Spacer,
+  HalfButton,
+} from './Modal.styled';
+import ProtonSDK from '../../services/proton';
+import { formatPrice } from '../../utils';
+import { ReactComponent as CloseIcon } from '../../public/close.svg';
+
+export const ClaimModal = (): JSX.Element => {
+  const {
+    currentUser,
+    currentUserBalance,
+    updateCurrentUserBalance,
+  } = useAuthContext();
+  const { closeModal } = useModalContext();
+  const [error, setError] = useState<string>('');
+
+  useEffect(() => {
+    if (error) setError('');
+  }, []);
+
+  const withdraw = async () => {
+    try {
+      const res = await ProtonSDK.withdraw({
+        actor: currentUser ? currentUser.actor : '',
+        amount: `${currentUserBalance}`,
```

Comment:
No need for interpolation. Also, same comment as in the banner component - we need to use the atomic market balance instead of `currentUserBalance`.